text/0000-artifact-hub-integration.md

```diff
@@ -0,0 +1,339 @@
+# Meta
+
+[meta]: #meta
+
+- Name: ArtifactHub.io Integration
+- Start Date: 2024-01-16
+- Author(s): @joeybrown-sf
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+
+[summary]: #summary
+
+This RFC proposes that the Buildpacks project integrates and augments the
+bespoke [registry service](https://registry.buildpacks.io/) with [ArtifactHub.io](https://artifacthub.io/) and
+eventually deprecates the buildpack registry.
+
+This RFC proposes that in 12 months, the buildpacks registry index becomes read-only. Buildpack publishers
+will have 12 months to change their publish process so that buildpacks are no longer published to the registry index,
```

Comment:
hmm, will they actually need to change their publish process, or could our tooling (i.e. `pack`) paper over the changes so that they go forward not worrying about the backend change?

I suppose they would have to change their expectations about the website and slack notifications.

Reply:
ok, so we're saying existing users of Buildpack Registry will need to take a one-time action (step 2)?

Reply:
yes that is correct. They will need to claim ownership of their artifacts in the ArtifactHub.io management console.

Reply:
Do you have any idea about how long the delay is for steps 3 & 4? Just wondering if we're talking a couple of minutes or a couple of hours or longer?